x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/integration_tests/response_actions_log.test.tsx

```diff
@@ -470,7 +470,7 @@ describe('Response actions history', () => {
       );
     });
 
-    it('should expand each row to show details', async () => {
+    it('should expand/collapse each row to show/hide details', async () => {
```

Comment:
Just renaming this test here. This tests the expand/collapse on the component/flyout.